Improve instructions for creating a dev branch for a Trio fork

marionbarker opened the pull request:
## Purpose
 
Insert missing instructions to assist people creating a dev branch from nightscout/Trio.
While doing this - add the trick of adding "> " to indicate figures go with text.
Add instructions for someone who has never built Trio, i.e., coming from Loop or iAPS
Add information about the `DONT RUN` or `DO NOT RUN` actions, suggest they be disabled

## Background

When I copied from loopdocs, I meant to create and insert new figures for Trio, but I forgot to do it.

I have seen several people struggling with this step and when I looked at the triodocs it was obvious why it was an issue.

marionbarker committed:
add graphic to assist in adding dev branch

Stampertina reviewed (commented):
One little thing: 
![image](https://github.com/user-attachments/assets/183fb4db-2309-4cff-a942-273ab2f44ef9)

marionbarker committed:
fix typo

ebouchut committed:
fix(install): Fix broken link in "Update with Browser"

marionbarker committed:
add instructions for new Trio builder, clarify some information

MikePlante1 commented on docs/install/build/browser/update-with-browser.md:
```diff
@@ -16,10 +21,22 @@ This temporary set of instructions is intended for the subset of users who **pre
+!!! important "Experienced Mac-Xcode Builder"
+    If you are an experienced Trio 0.2.x user who wants to join the open beta testing, but also wants to switch from Mac-Xcode build to Browser Build - welcome. For the time being, please do this:
+
+    * Use the [0.2.x: Build Trio with *GitHub*](../../../0.2.x/operate/build.md#build-trio-with-github){: target="_blank"} instructions to build Trio 0.2.5 and wait for it to show up in your TestFlight. Do not install it - you just want to make sure you can succeed with a Browser Build.
```
Probably shouldn't specify 0.2.5 in case 0.2.6 is released before 1.0 is. 
Maybe "to build the latest Trio 0.2.x"

ebouchut committed:
style(install): Fix headings level

Change level 3 headings to level 2

ebouchut commented on docs/install/build/browser/update-with-browser.md:
```diff
@@ -1,12 +1,17 @@
+        * They can be customized by you if required, reach out for help on how to do this
+    * **Your saved Temp Targets and Overrides will not be maintained**
+        * The storage method in `0.5.x` is different from `0.2.x`, so you will need to recreate them
+        * Capture a screenshot of each important TT or Override that you want to add to `0.5.x`
```
I suggest replacing `TT` with `Temporary Target (TT)` or `Temporary Target`.

ebouchut commented on docs/install/build/browser/update-with-browser.md:
```diff
@@ -16,10 +21,22 @@ This temporary set of instructions is intended for the subset of users who **pre
+    If you are an experienced Trio 0.2.x user who wants to join the open beta testing, but also wants to switch from Mac-Xcode build to Browser Build - welcome. For the time being, please do this:
+
+    * Use the [0.2.x: Build Trio with *GitHub*](../../../0.2.x/operate/build.md#build-trio-with-github){: target="_blank"} instructions to build Trio 0.2.5 and wait for it to show up in your TestFlight. Do not install it - you just want to make sure you can succeed with a Browser Build.
+    * Then return to this page and follow the directions; you will need to run Add Identifiers and Create Certificates again when updating to 0.5.x.
```
We need to find a way to prevent *Google* from automatically translating the names of GitHub Actions `Add Identifiers` and `Create Certificates.` Perhaps we can **move** them **to the very end of the sentence** and **enclose them** in `<code>` tag (`<code>Add Identifiers</code>`) / backticks (`` `Create Certificates` ``).

marionbarker replied:
I made a change. If this is not sufficient, I'll try again.

dnzxy commented on docs/install/build/browser/update-with-browser.md:
```diff
@@ -1,12 +1,17 @@
-    * If you choose to downgrade, you will need to set everything up again from scratch.
+    * **Once you have upgraded to `0.5.x`, going back to `0.2.x` is not supported**
+        * If you choose to downgrade, you will need to set everything up again from scratch
+    * Some guardrails may be more limiting that `0.2.x`
```
Remove this bulletpoint please. It's an ongoing discussion, with HCPs and community members. This ought to not be part of docs to be removed later. We will not introduce the narrative of guardrails being a bad thing that needs removal into our docs.

dnzxy reviewed (changes requested):
Overall, looks good. However please remove the highlighted bullet point for now. Thank you.

ebouchut committed:
fix(install): Fix a broken link on the "Update with Browser" page

ebouchut committed:
style(install): Add bold to make some key words stand out

ebouchut committed:
fix(install): Fix typos

I may go overboard with this sometimes — just trying to keep my G7y buddy happy!
It really likes commas ;-)

ebouchut commented on docs/install/build/browser/update-with-browser.md:
```diff
@@ -149,7 +176,7 @@ If you completed all the steps on this page successfully (got a green checkmark
 If you have one branch as default, for example `main`, and choose to build a different branch, there is an extra step when you `Build Trio`. In addition to the normal steps 1, 2 and 3 in the graphic below, you must also do the (optional) step. Select the `dev branch` in the `branch dropdown` menu before continuing to step 4 and tapping on the green Run workflow button:
 
-![build app using github actions](img/action-04-build-trio.svg){width="700"}
+> ![build app using github actions](img/action-04-build-trio.svg){width="700"}
```
Suggestion: `github actions` => `GitHub Actions`

marionbarker replied:
Does alt text get automatically translated? What are the rules we want to apply to alt text?
Let's hold this change for next time.

ebouchut committed:
style(install): Add a line break and enclose identifiers with code tag

in the `Update with Browser` page

MikePlante1 reviewed (commented):
Not a blocker, just a suggestion:

ebouchut committed:
style(install): Add inline code for GH Action, list and indentations

ebouchut reviewed (dismissed):
👍 I left a few suggestions (not blocker), but this LGTM.

marionbarker committed:
reword some update compatibility notes

marionbarker committed:
use 0.2.x, not 0.2.5
